ask-sdk-model-runtime/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "ask-sdk-model-runtime",
+    "version": "1.0",
+    "description": "The base components for ASK SDK Models",
+    "main": "index.js",
+    "types": "index.d.ts",
+    "scripts": {
+      "gulp": "./node_modules/.bin/gulp"
+    },
+    "author": "Amazon.com",
+    "contributors": [
+      {
+        "name": "ASK SDK",
+        "email": "ask-sdk@amazon.com"
+      }
+    ],
+    "license": "Apache-2.0",
+    "keywords": [
```

Comment:
You should probably include models and model runtime in this list

Reply:
Got it, will add